include/modules/sv_detecting_methods/analyze_sa_tag_method.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "structures/aligned_segment.hpp"   // for struct AlignedSegment
+
+/*! \brief Splits a string by a given delimiter and stores substrings in a given container.
+ *
+ * \param str   string to split
+ * \param cont  container for the splitted substrings
+ * \param delim delimiter
+ */
+template <class Container>
+void split_string(const std::string& str, Container& cont, char delim = ' ');
+
+/*! \brief Parse the SA tag from the SAM/BAM alignment of a chimeric/split-aligned read. Build
+ *         [aligned_segments](\ref AlignedSegment), one for each alignment segment of the read.
+ *
+ * \param sa_string         "SA" tag string
+ * \param aligned_segments  vector of [aligned_segments](\ref AlignedSegment).
```

Comment:
There are no full stops after any other param so I assume there shouldn't be one here ? :)
```suggestion
 * \param aligned_segments  vector of [aligned_segments](\ref AlignedSegment)
```